tpl/default/linklist.html

```diff
@@ -136,12 +136,10 @@
             {if="$thumbnails_enabled && !empty($value.thumbnail)"}
               <div class="linklist-item-thumbnail" style="width:{$thumbnails_width}px;height:{$thumbnails_height}px;">
                 <div class="thumbnail">
-                  <a href="{$value.real_url}">
```

Comment:
I'm not in favor of dropping the link on thumbnails. Yes, it's identical to the link on titles, but I find myself using it quite often. Maybe because the click area is bigger, and they're aligned on the page.

Reply:
On a second thought I agree, I'm not sure we should remove the link on the thumbnail, many people must be used to clicking on it.

Reply:
OK, no problem.

I suggest then to keep the link but hide it for screenreader users and keyboard users.

```suggestion
                  <a href="{$value.real_url}" aria-hidden="true" tabindex="-1">
```

Reply:
Sure, go ahead. :)